plugins/modules/pipx.py

```diff
@@ -26,13 +26,31 @@
 options:
     state:
         type: str
-        choices: [present, absent, install, uninstall, uninstall_all, inject, upgrade, upgrade_all, reinstall, reinstall_all, latest]
+        choices:
+            - present
+            - absent
+            - install
+            - install_all
+            - uninstall
+            - uninstall_all
+            - inject
+            - uninject
+            - upgrade
+            - upgrade_shared
+            - upgrade_all
+            - reinstall
+            - reinstall_all
+            - latest
+            - pin
+            - unpin
         default: install
         description:
             - Desired state for the application.
             - The states V(present) and V(absent) are aliases to V(install) and V(uninstall), respectively.
             - The state V(latest) is equivalent to executing the task twice, with state V(install) and then V(upgrade).
               It was added in community.general 5.5.0.
+            - The states V(install_all), V(uninject), V(upgrade-shared), V(pin) and V(unpin) are only available in C(pipx>=1.6.0),
+              make sure to have a compatible version when using this option.
```

Comment:
You should also mention that the new states were added in community.general 9.4.0.

Reply:
done